File: Easy/0389-Find the Difference/Count.rs
impl Solution {
    pub fn find_the_difference(s: String, t: String) -> char {
        let mut cnt = [0; 26];

        for ch in s.bytes() {
            cnt[(ch - b'a') as usize] += 1;
        }

        for ch in t.bytes() {
            cnt[(ch - b'a') as usize] -= 1;
            if cnt[(ch - b'a') as usize] == -1 {
                return ch as char;
            }
        }

        ' '
    }
}

File: Easy/0762-Prime Number of Set Bits in Binary Representation/Count.py
class Solution:
    def countPrimeSetBits(self, L: int, R: int) -> int:
        cnt = 0

        for i in range(L, R + 1):
            if bin(i).count('1') in {2, 3, 5, 7, 11, 13, 17, 19}:
                cnt += 1

        return cnt
